exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterTest.java

```diff
@@ -244,4 +249,55 @@ void stringRepresentation() {
               "ZipkinSpanExporter{endpoint=http://zipkin:9411/api/v2/spans, compressionEnabled=false, readTimeoutMillis=15000}");
     }
   }
+
+  @Test
+  void testSuppressInstrumentation() {
```

Comment:
I know we are consistent, but jupiter doesn't need the 'test' prefix

```suggestion
  void suppressInstrumentation() {
```